### ISSUE TITLE
[CoE Starter Kit - BUG]Upgrade core Solution from 3.41 to 4.02 

### ISSUE THREAD

**Issue by Raz-786** [microsoft/coe-starter-kit#4638]
### Describe the issue

during import Core solution from 3.41 to 42. i am getting an error 

"You'll need to import the following items to this environment in order to import Center of Excellence - Core Components"

Creator Kit Core, this is new to me never in past up until when we upgrade COE

any help would be greatly appreciated

thanks

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

3.41

### What app or flow are you having the issue with?

Core Solution during import

**Comment by manuelap-msft** [comment]
Yes, a new dependency on the Creator Kit was introduced with our December release. 

Please see the new pre-requisite mentioned in our release note
https://github.com/microsoft/coe-starter-kit/releases/tag/CoEStarterKit-December2022

**Comment by Raz-786** [comment]
Thank you

Also is there any process you are aware of before upgrade the solution, I can backup and in case if there is any issue during upgrade process then I do have option to revert back my instance with previous version of core solution

Thanks

**Comment by Jenefer-Monroe** [comment]
People often use a dev/test environment for testing upgrade. 
But other than that we dont have a recommended process for upgrade aside from what is documented. [Updating the Center of Excellence (CoE) Starter Kit](https://learn.microsoft.com/power-platform/guidance/coe/after-setup)

**Comment by Raz-786** [comment]
Thanks Jenefer,

I do have test environment and currently not functional, if I just to test the upgrade process should I disable "Enable Plugin steps and flow included in the solution" see below screen shot, because I don't want to flow enable and sending an email from both environment i.e. Prod  and Dev

Sorry I am very new to this

***@***.***
Thanks

**Comment by Jenefer-Monroe** [comment]
Hello please note that screenshots will not come through if you send them in email. You will need to log in to GitHub to share pictures. 

We do not suggest you change any settings like that. Instead note that we have an environment variable called **_ProductionEnvironment_**, if set to "no" (how you should set test envt for example) then all emails/chats/etc go to the admin .
Please see [All environment variables](https://learn.microsoft.com/power-platform/guidance/coe/setup-core-components#all-environment-variables)
![image](https://user-images.githubusercontent.com/89584709/213819417-d10e1543-a290-4eff-b958-3199bebec046.png)

Lastly, no problem, we are here to help!

**Comment by Raz-786** [comment]
Thanks Jenefer,

I am just wondering if you step by step installation along with screen shot, reason for asking I can't change "ProductionEnvironment" variable to "No" for this I need Command Center app needs to be install but don't know how to do that


Thanks

**Comment by Jenefer-Monroe** [comment]
If you look further down in the [update environment variables](https://learn.microsoft.com/power-platform/guidance/coe/faq#update-environment-variables) section you will see this part where it described setting env vars without the command center.

**Comment by Raz-786** [comment]
Thanks Jenefer

Yes, Your are right I found the step to change the ProductionEnvironment value to No based on below step
If you aren't using the Admin - Command Center<https://learn.microsoft.com/en-us/power-platform/guidance/coe/core-components#admin---command-center> app, do the following to update environment variables:
1.    If you've installed the solution in a production environment, do the following:
a.    Go to Power Automate<https://make.powerautomate.com/>.
b.    On the left pane, select Solutions.
c.     Select the Default Solution, and change the filter to show Environment Variables.
d.    Select a variable that you want to update, and then configure its Current Value.

But after changing the value there is special notes in the document

After changing the value of an environment variable, you'll need to turn off and on all the flows that use that environment variable to make sure the flows use the latest value.

How do I know which flow using that environment variable?

Thanks

**Comment by Jenefer-Monroe** [comment]
That step is no longer needed, the product team fixed that issue and now the flows update with the env var values. Yah!
